### PR TITLE
Update for review comments including small writes

### PR DESCRIPTION
**Pull request by bremoran**
* Rename "run" to "invoke
* Rename "common sequence" to "shared sequence"
* Add clarifying text about Run Sequence when Strict Order = False
* Clarify that "suit-invoke" sequence may also contain image checks if necessary
* Add direct-write command and parameter, and content check condition
* Add clarifying text explaining stop conditions for try-each

### CODE REVIEW

File: draft-ietf-suit-manifest.cddl [hunk 1]
```diff
@@ -85,25 +85,25 @@ suit-cose-hash-algs /= cose-alg-shake256
 
 SUIT_Component_Identifier =  [* bstr]
 
-SUIT_Common_Sequence = [
+SUIT_Shared_Sequence = [
     + ( SUIT_Condition // SUIT_Common_Commands )
```
Comment (oyvindronningstad) [inline draft-ietf-suit-manifest.cddl:89]: Should "Common_Commands" be "Shared_Commands" as well?

**Review by oyvindronningstad — COMMENTED** [review]
Was the PR supposed to contain changes to the .md file as well?

File: draft-ietf-suit-manifest.cddl (hunk 1 above)
Comment (bremoran) [reply draft-ietf-suit-manifest.cddl:89]: Yes, it probably should.

File: draft-ietf-suit-manifest.md [hunk 2]
```diff
@@ -548,7 +556,14 @@ When the manifest processor encounters any of these scenarios the parallel proce
...
+To perform more useful parallel operations, a manifest author may collect sequences of commands in a Run Sequence command. Then, each of these sequences MAY be run in parallel. There are several invocation options for Run Sequence:
+
+* Component Index is a positive integer, Strict Order is False: Strict Order is set to True before the sequence argument is run. The sequence argument MUST begin with set-component-index.
+* Component Index is True or an array of positive integers, Strict Order is False: The sequence argument is run once for each component (or each component in the array) the component index is set by the interpreter without a directive, and Strict Order is set to True before each iteration.
```
Comment (dthaler) [inline draft-ietf-suit-manifest.md:562]: the CDDL has "true" not "True" for Component Index, should this be "true" like in line 1224?
```suggestion
* Component Index is true or an array of positive integers, Strict Order is False: The sequence argument is run once for each component (or each component in the array) the component index is set by the interpreter without a directive, and Strict Order is set to True before each iteration.
```
Also I am having a hard time parsing grammar around the phrase "the component index is set by the interpreter without a directive", can you reword?  Same in 564

File: draft-ietf-suit-manifest.md [hunk 3]
```diff
@@ -1018,6 +1034,14 @@ The size of the firmware image in bytes. This size is encoded as a positive inte
 
 This parameter sets the slot index of a component. Some components support multiple possible Slots (offsets into a storage area). This parameter describes the intended Slot to use, identified by its index into the component's storage area. This slot MUST be encoded as a positive integer.
 
+#### suit-parameter-content {#suit-parameter-content}
+
+A block of raw data for use with {{suit-directive-write}}. Contains a byte string of data to be written to a specified component ID in the same way as a fetch or a copy.
```
Comment (dthaler) [inline draft-ietf-suit-manifest.md:1039]:
```suggestion
A block of raw data for use with {{suit-directive-write}}. It contains a byte string of data to be written to a specified component ID in the same way as a fetch or a copy.
```

**Review by oyvindronningstad — COMMENTED** [review]
There are still a few instances of "common" in the cddl.

**Comment by bremoran** [comment]
> common

Were you thinking that suit-common (the container for components, and suit-shared-commands) should be renamed "suit-shared"?

**Comment by oyvindronningstad** [comment]
> Were you thinking that suit-common (the container for components, and suit-shared-commands) should be renamed "suit-shared"?

Not necessarily, it's up to you. There's `SUIT_Directive_Try_Each_Argument_Common` though, which should probably be `SUIT_Directive_Try_Each_Argument_Shared`

**Comment by bremoran** [comment]
Yup, I've made that change locally and was just holding off until I got input on whether there was a need to rename suit-common.

**Pull request merged**